Challenge1.exs

```diff
@@ -0,0 +1,40 @@
+defmodule Challenge do
+  def num_divisible_by_3_sequence(a, b) do
+    if a >= 1 and a <= b and b < 231 do
+      sequence = ["1"]
+      ini = 1
+      con = 0
+      sequence = construct_sequence(sequence, ini, a, b, con)
+      IO.inspect(sequence)
+      IO.puts("Number of divisibles by 3 elements in the sequence: #{Enum.reduce(Enum.filter(sequence, fn element -> rem(String.to_integer(element), 3) == 0 end), 0, fn _element_, acc -> acc + 1 end)}")
```

Comment:
Instead of using reduce you can also use a length function call which is easier to read :) 🧠 , also remember underscore-starting variables are only that way, starting, when they are not intended to be used (so it could be `_element` instead of `_element_`)